server/src/main/java/csearch/model/CsProject.java

```diff
@@ -0,0 +1,67 @@
+package csearch.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "csprojects")
+public class CsProject {
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private int id;
+  private String title;
+  private String description;
+  private String process;
+  private int difficulty;
+
+  public CsProject() {
+    setTitle("");
```

Comment:
You should use 
```
title = "";
description = "";
```
usually setTitle or setSomething("abc") is used outside of the class. That's why we make it a public get/set. Within a class, you can access it easily by just calling the variable